src/controllers/profile.controller.ts

```diff
@@ -8,3 +10,69 @@ export const getProfile = async (req: Request, res: Response) => {
     res.status(500).json({ error: err });
   }
 };
+
+export const updateProfile = async (req: Request, res: Response) => {
+  try {
+    const user: any = req.user;
```

Comment:
Let's avoid using any 

```suggestion
    const user: Profile = req.user;
```

Reply:
I'm unable to change the data type for this other than any..not sure why. Any solution?